README.md

```diff
@@ -83,8 +83,47 @@ confidence in them.
 
 ## Measurement modes
 
-Tachometer currently supports three kinds of time interval measurements,
-controlled with the `--measure` flag.
+Tachometer supports four kinds of time interval measurements, controlled with
+the `measurement` config file property, or the `--measure` flag.
+
+#### Performance API
+
+Retrieve a measure, mark, or paint timing from the
+[`performance.getEntriesByName`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntriesByName)
+API. Note this mode can only be used with a config file.
+
+For example, in your benchmark:
+
+```javascript
+performance.mark('foo-start');
+// Do some work ...
+performance.mark('foo-stop');
+performance.measure('foo', 'foo-start', 'foo-stop');
+```
+
+And in your config file:
+
+```json
+"benchmarks": [
+  {
+    "measurement": {
+      "performanceEntry": {
+        "name": "foo"
+      }
+    }
+  }
+]
+```
+
+The following performance entry types are supported:
+
+- [`measure`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceMeasure):
```

Comment:
Consider mentioning that `measure` is good for benchmarking a specific bit of code, while `mark` is good for measuring the total page load time up to a certain point

Reply:
Done.